app/views/shared/check_answers/_assets.html.erb

```diff
@@ -3,73 +3,68 @@
 <% online_savings_accounts = @legal_aid_application.online_savings_accounts_balance %>
 <% online_current_accounts = @legal_aid_application.online_current_accounts_balance %>
 
-<section class="print-no-break">
-  <%= render "shared/check_answers/property", read_only:, individual: %>
-</section>
+<%= render "shared/check_answers/property", read_only:, individual: %>
 
-<section class="print-no-break">
-  <%= render "shared/check_answers/vehicles", read_only:, individual: %>
+<%= render "shared/check_answers/vehicles", read_only:, individual: %>
 
-  <h2 class="govuk-heading-l"><%= t(".assets.bank_accounts_heading") %></h2>
+<h2 class="govuk-heading-l print-no-break-after"><%= t(".assets.bank_accounts_heading") %></h2>
 
   <% if @legal_aid_application.non_passported? && !@legal_aid_application.client_uploading_bank_statements? %>
     <!-- non-passported truelayer only -->
 
     <% if read_only %>
-      <%= govuk_summary_card(title: t(".assets.clients_bank_accounts"),
-                             heading_level: 3,
-                             html_attributes: { id: "applicant-bank-accounts-details" }) do |card| %>
-        <% unless read_only %>
-          <% card.with_action do %>
-            <%= govuk_link_to(t("generic.change"),
-                              check_answer_url_for(journey_type, :offline_accounts, @legal_aid_application),
-                              "aria-label": "#{t('generic.change')} #{t('.assets.client_bank_accounts')} values") %>
+        <%= govuk_summary_card(title: t(".assets.clients_bank_accounts"),
```

Comment:
I was trying to work out why there is so much of a diff on this file. Is it because you've accidentally double indented on line 16?